extensions/modules/src/org/exist/xquery/modules/image/CropFunction.java

```diff
@@ -132,11 +132,10 @@ public Sequence eval(Sequence[] args, Sequence contextSequence) throws XPathExce
         String formatName = mimeType.substring(mimeType.indexOf("/")+1);
 
         //TODO currently ONLY tested for JPEG!!!
-        Image image = null;
         BufferedImage bImage = null;
         try {
             //get the image data
-            image = ImageIO.read(((BinaryValue)args[0].itemAt(0)).getInputStream());
+            Image image = ImageIO.read(((BinaryValue)args[0].itemAt(0)).getInputStream());
```

Comment:
This `InputStream` needs to be closed when it is finished with.

From the javadoc:

> This method <em>does not</em> close the provided
> <code>InputStream</code> after the read operation has completed;
> it is the responsibility of the caller to close the stream, if desired.